proto/storage/external_backup.proto

```diff
@@ -35,6 +37,27 @@ message S3Config {
   string endpoint = 7; // @gotags: scrub:"dependent" validate:"nolocalendpoint"
 }
 
+enum S3URLStyle {
+  S3_URL_STYLE_UNSPECIFIED = 0;
+  S3_URL_STYLE_VIRTUAL_HOSTED = 1;
+  S3_URL_STYLE_PATH = 2;
+}
+
+message S3Compatible {
```

Comment:
Let's add a comment how `S3Compatible` differs from `S3` (non-AWS providers like minio vs official AWS S3).

Reply:
Added a comment. Let me know what you think.

Reply:
I think the text could be refined slightly, see my suggestions above.